.github/workflows/draft_new_release.yml

```diff
@@ -0,0 +1,77 @@
+name: "Draft new release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The new version in X.Y.Z format."
+        required: true
+
+jobs:
+  draft-new-release:
+    name: "Draft a new release"
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      RELEASE_BRANCH: release/${{ github.event.inputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.TENTENONE_GITHUB_TOKEN }}
+
+      - name: Create release branch
+        run: git checkout -b ${{ env.RELEASE_BRANCH }}
+
+      - name: Initialize mandatory git config
+        run: |
+          git config user.name "${{ secrets.TENTENONE_NAME }}"
+          git config user.email ${{ secrets.TENTENONE_EMAIL }}
```

Comment:
What is this needed for?

Reply:
copied from itchysats, I guess its for the commit to have a proper format.